docs/guides/high-availability/connect-keycloak-to-external-infinispan.adoc

```diff
@@ -0,0 +1,79 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Connect {project_name} with an external {jdgserver_name}"
+summary="Building block for an Infinispan deployment on Kubernetes"
+preview="true"
+tileVisible="false" >
+
+This topic describes advanced {jdgserver_name} configurations for {project_name} on Kubernetes.
```

Comment:
One aspect that should also be considered when using an external Infinispan cluster is that the current readiness probes only check for a working database connection. However, an additional test for a working Infinispan connection would be useful here.

Reply:
We have the following issue for this: https://github.com/keycloak/keycloak-benchmark/issues/591 and hope to fix it soon. When you look for "Infinispan cluster failure" in "concepts-active-passive-sync.adoc" you'll see it listed there that human intervention is needed.